lib/App/DuckPAN/Fathead.pm

```diff
@@ -0,0 +1,257 @@
+package App::DuckPAN::Fathead;
+# ABSTRACT: Searches a given output.txt file for a query match
+
+use Moo;
+with 'App::DuckPAN::HasApp';
+
+use DBI;
+use JSON;
+use Path::Tiny;
+use HTML::TreeBuilder;
+use HTML::Element;
+use Data::Printer return_value => 'dump';
+
+has selected => (
+	is => 'rw',
+	lazy => 1,
+	required => 0,
+	predicate => 1,
+	trigger => 1
+);
+
+sub _trigger_selected {
+	my ( $self, $id ) = @_;
+	my $dir = path("lib/fathead/$id");
+	unless ($dir->is_dir) {
+		my $full_path = $dir->realpath;
+		$self->app->emit_and_exit(1, "Directory not found: $full_path") ;
```

Comment:
@jbarrett I actually check below in the `output_txt` builder if the file exists. I suppose I could do that check in here and define `output_txt` here as well, removing the need for the builder?

Looking back, I don't recall why I used a builder for `output_txt`